src/index.html

```diff
@@ -20,8 +20,692 @@
       href="styles/index.scss"
     />
   </head>
-
   <body>
-    <h1>Catalog</h1>
+    <div class="container">
+      <header class="header">
+        <div class="header__box">
+          <a
+            class="header__logo"
+            href="index.html"
+          >
+            <img
+              src="./images/logo.png"
```

Comment:
Consider providing a more descriptive `alt` text for the logo image to improve accessibility. Instead of 'logo', use something like 'Company Logo'.